source/grid/grid_generator.cc

```diff
@@ -4990,6 +4990,354 @@ namespace GridGenerator
   }
 
 
+  void non_standard_orientation_mesh_2D(Triangulation<2> &tria,
```

Comment:
You'll have to rename it here as well in the .cc file or you'll get linker errors.